.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: c
+sudo: required
+dist: trusty
+matrix:
```

Comment:
I guessed that you'd have an internal CI. However, eventually people (like me!) might want to contribute to zrpcd. To help with that, I suggest that you consider moving your development towards an open model, use the github repo as the authoritative repo instead of any internal ones, move the CI to this repo, add contribution guidelines, etc etc.
This will ease the process both for you and contributors, the project will seem more active, more attractive to users and contributors etc, but of course implementing such a thing and seeing the benefits of this will take time.

Wrt the point about other distros, yes that can come later. I didn't include this in this PR because travis offers only Ubuntu and CentOS would have to be done using containers and I wanted to keep the changes as small as possible.

Anyway, thanks for your time helping with my PRs :-)